### ISSUE TITLE
Styles are not being generated without HMR

### ISSUE THREAD

**Issue by www-chique** [windicss/vite-plugin-windicss#206]
**Describe the bug**
When using Svelte's class syntax, the styles are not being loaded.
Example: **class:focus:underline** in _src/components/Link.svelte_
```
<a {href} class="font-semibold no-underline cursor-pointer" on:click class:focus:underline={href} class:hover:underline={href} tabindex="0">
	<slot />
</a>
```
Additionally, in other places a few classes doesn't generate styles randomly. But I couldn't reproduce those in a minimal repo. I am trying to use "z-20" or "h-[52px]" for example, and it doesn't generate until I go into that component and cut and paste the class, save the file, triggering the hot reload.

This might be similar to #91 , but I followed the instructions given there, and added the scan.dirs option. But that didn't work.

**Minimal Reproductions**
Here is a minimal repo: [Sample repo](https://github.com/www-chique/windi-class)
This is based on: [Routify Vite Typescript starter template](https://github.com/lamualfa/routify-vite-ts)

**Versions**
- vite-plugin-windicss: 1.1.1
- vite: 2.2.3
- framework:  Svelte 3.37.0 + [Routify](routify.dev/)

**Additonal Context**
In actual production repo, there's a lot of other components and things involved, so a few other classes doesn't seem to work, like the "z-20" or "h-[52px]" examples I mentioned above. But couldn't recreate them in the minimal repo.
Another interesting thing to note is - 
Previously I was on **vite-plugin-windicss version 0.15.10**, and also used svelte-windicss-preprocess 3.5.3. That combination still works for me just fine. However, I wanted to update the dependencies to the latest, and found out that I do not need the svelte-windicss-preprocess if I just use the vite-plugin-windicss. And since then I am facing this **cold start** issue. This happens during build time as well. Building the project doesn't output those classes. I don't know why it's only affecting a few classes, whereas some others are just working fine.

I am a huge fan of WindiCSS, and I absolutely love it over anything else. The modifier grouping and everything is on point. It's the best thing out there. Many thanks to all of you guys.
If anyone with a little idea can point me to the right direction, I'll be grateful.

**Comment by antfu** [comment]
I'd see where the `"z-20" or "h-[52px]"` comes from, if you are generate them dynamically, it probably won't work.

Related with this?
https://windicss.org/guide/extractions.html

**Comment by www-chique** [comment]
@antfu - I appreciate you looking into this. I have even added the following extract option in the config, but it still wouldn't work. Is there something I am missing? I am aware I can keep adding those to a "safelist", but that doesn't look like an elegant solution.

```
extract: {
    // accepts globs and file paths relative to project root
    include: [
      'index.html',
      'src/**/*.{svelte,html,jsx,tsx}',
    ],
    exclude: [
      'node_modules/**/*',
      '.git/**/*',
    ],
  },
```

P.S. As I mentioned I can still work with the older version of this plugin, that serves the purpose for me. But thought this issue might shed some light into some migration instructions I have missed.

**Comment by antfu** [comment]
should be fix in c3cc80d6b28bda9c62a06a159feebead2f920fbc
